### DISCUSSION TITLE
Refresh rate not toggling properly from 165 to 60 ASUS M16 2021 GU603HM

### DISCUSSION THREAD

**Discussion by dark-trojan789** [seerge/g-helper#2946] [Q&A]
Hello, whenever the refresh rate is changed from ghelper from 165 to 60 hz it only changes desktop mode from 165 to 60 but active signal is still 165 but when i change from settings there is an option of 60.03 hz which properly configures the refresh rate and the screen goes off and comes back on the same way it used to in asus armourary crate. how to fix this? thank you
![Screenshot 2024-08-08 163905](https://github.com/user-attachments/assets/357f8b8a-b0aa-4606-90e6-bd4a1ab64d68)
![Screenshot 2024-08-08 163853](https://github.com/user-attachments/assets/c203714f-fa3b-47a7-8a11-fcee77c1c074)

**Comment by seerge** [comment]
@dark-trojan789 G-Helper changes refresh rate via the only possible method - WindowsAPI :) How exactly it happens depends on windows and GPU drivers.

**Reply by seerge** [reply]
P.S. My G14 2022 (AMD version) also behaves like that, due to a recent changes in AMD drivers.

As an experiment I just installed [Asus Smart Display Control](https://dlcdnets.asus.com/pub/ASUS/GamingNB/Image/Software/SoftwareandUtility/15237/ASUSSmartDisplayControl_V2.7.18_15237_1.zip) app again to check if it's behavior is different from what G-Helper can do. And it's exactly same :)
